sql/moz-fx-data-shared-prod/monitoring_derived/bigquery_usage_v2/query.sql

```diff
@@ -0,0 +1,100 @@
+{% set DEFAULT_PROJECTS = [
+    "mozdata",
+    "moz-fx-data-shared-prod",
+    "moz-fx-data-marketing-prod",
+] %}
+
+WITH jobs_by_org AS (
+  SELECT
+    t1.project_id AS source_project,
+    creation_date,
+    job_id,
+    job_type,
+    reservation_id,
+    cache_hit,
+    state,
+    statement_type,
+    referenced_tables.project_id AS reference_project_id,
+    referenced_tables.dataset_id AS reference_dataset_id,
+    referenced_tables.table_id AS reference_table_id,
+    destination_table.project_id AS  destination_project_id,
+    destination_table.dataset_id AS  destination_dataset_id,
+    destination_table.table_id AS  destination_table_id,
+    user_email,
+    end_time-start_time as task_duration,
+    ROUND(total_bytes_processed / 1024 / 1024 / 1024 / 1024, 4)
+    AS total_terabytes_processed,
+    ROUND(total_bytes_billed / 1024 / 1024 / 1024 / 1024, 4)
+    AS total_terabytes_billed,
+    total_slot_ms,
+    error_result.location AS error_location,
+    error_result.reason AS error_reason,
+    error_result.message AS error_message,
+    query_info_resource_warning AS resource_warning,
+  FROM
+    `moz-fx-data-shared-prod.monitoring_derived.jobs_by_organization_v1` AS t1
+  LEFT JOIN
+    UNNEST(referenced_tables) AS referenced_tables
+  ),
+  {#- format off #}
```

Comment:
I know the SQL-with-Jinja code example I showed you used `format off` and `format on` directives to allow manual formatting of the Jinja portion of the query, but in general I'd suggest trying to live with the default formatting and only using manual formatting when the auto-formatting is making the code less readable, as the auto-formatting helps keep our code consistent.

Specifically, in this case I'd suggest:
1. Remove the `format off` and `format on` directives.
2. Run `bqetl format` on this file.
3. If you really don't like how it formatted this section, put the `format off` and `format on` directives back and tweak the formatting as desired.

Reply:
I've removed the `format off` and `format on` directives